### ISSUE TITLE
Add VSCode to list of text editors

### ISSUE THREAD

**Issue by scottschup** [railsbridge/docs#666]
VSCode may be a little weightier than the average new-to-coding individual needs, but once you get familiar with it, it's pretty great to have the built in terminal and all of the cool features that make debugging, linting, and navigating the code base easier. And it's especially great that it is compatible with so many languages and frameworks. I think it's worth adding to the list, even if it's not the default recommendation.

**Comment by LautaroLobo12** [comment]
Yep, I think it's worth adding it to the list too. Uhm, maybe someone from the organization can share their thoughts? That would be cool, if enough people agree I could make a PR.

_oh dang this issue is old. Still think that VS Code should be added_
